examples/api/test/widgets/indexed_stack/indexed_stack.0_test.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This file should be located at `examples/api/test/widgets/basic/indexed_stack.0_test.dart`.